README.md

```diff
@@ -338,46 +350,69 @@ Append the first activity to occur after the primary activity.
 
 **Example Usage:**
 
-For the first 'visited_website' append **First After** 'signed_up'. For each
+For the **First Ever** 'visited_website' append **First After** 'signed_up'. For each
 customer add whether or not they converted any time after their first visit to
 the site.
 
+> **Note:** Be catious when using this with **All Ever** for the primary activity.
+> It will result in adding the same **First After** activity to multiple primary
+> activity records, if the appended activity occurred after multiple primary
+> activities. Consider using **First In Between** instead.
+
 ### Last After ([source](./macros/relationships/append_only/last_after.sql))
 Append the last activity to occur after the primary activity.
 
 **Dataset Usage:**
 - `primary_activity:` ❌
 - `appended_activity:` ✅
 
-**Example Usage:** For the **First Ever** 'visited_website' append **Last
+**Example Usage:**
+
+For the **First Ever** 'visited_website' append **Last
 After** 'returned_item. The most recent time a customer returned an item after
 their first visit.
 
-### First Before ([source](./macros/relationships/append_only/first_before.sql))
-Append the first activity to occur before the primary activity.
+> **Note:** Be catious when using this with **All Ever** for the primary activity.
+> It will result in adding the same **Last After** activity to multiple primary
+> activity records, if the appended activity occurred after multiple primary
+> activities. Consider using **Last In Between** instead.
+
+### First In Between ([source](./macros/relationships/append_only/first_in_between.sql))
+Append the first activity to occur after each occurrence of the primary
+activity, but before the next occurrence of the primary activity.
 
 **Dataset Usage:**
 - `primary_activity:` ❌
 - `appended_activity:` ✅
 
 **Example Usage:**
 
-For every 'visited_website' append **First Before** 'opened_email'. This will
-add the the first email that the customer opened before their first visit.
+For **All Ever** 'visited_website' append **First In Between** 'completed_order'. On
+every website visit, did the customer order before the next visit. (generally
+used for event-based conversion)
 
-### First In Between ([source](./macros/relationships/append_only/first_in_between.sql))
-Append the first activity to occur after each occurrence of the primary
-activity, but before the next occurrence of the primary activity.
+> **Note:** The appended activity will be added to the row of the final occurance of the
```

Comment:
Maybe edit this to `will _also_ be added` - since prior primary activity occurrences will also have appended activity records for this relationship if they exist.